<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiEditPlacemarkDialog</class>
 <widget class="QDialog" name="UiEditPlacemarkDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit placemark</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="Marble::PlacemarkEditHeader" name="m_header" native="true"/>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="m_descriptionTab">
      <attribute name="title">
       <string>Description</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QCheckBox" name="m_isFormattedTextMode">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Formatted text</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFontComboBox" name="m_fontFamily">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="m_fontSize">
           <property name="editable">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string>6</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>8</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>9</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>10</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>11</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>12</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>14</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>16</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>18</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>20</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>24</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>36</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>48</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>72</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>96</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QToolBar" name="m_formattedTextToolBar">
           <property name="windowTitle">
            <string>toolBar</string>
           </property>
           <addaction name="separator"/>
           <addaction name="m_actionBold"/>
           <addaction name="m_actionItalics"/>
           <addaction name="m_actionUnderlined"/>
           <addaction name="m_actionAddImage"/>
           <addaction name="m_actionAddLink"/>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="m_description"/>
       </item>
       <item>
        <widget class="QCheckBox" name="m_isBalloonVisible">
         <property name="text">
          <string>Show Balloon Popup</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="style_color_tab">
      <attribute name="title">
       <string>Style, Color</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Label</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Color:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="m_labelButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="iconSize">
               <size>
                <width>12</width>
                <height>12</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>13</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="m_labelScaleField">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Scale:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="m_labelScale">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="decimals">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>96</width>
              <height>17</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Icon</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QLabel" name="m_iconColorField">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="font">
               <font>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Color:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="m_iconButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="iconSize">
               <size>
                <width>12</width>
                <height>12</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_14">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>13</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QLabel" name="m_iconScaleField">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Scale:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="m_iconScale">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="decimals">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_15">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>96</width>
              <height>17</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_isPlacemarkVisible">
         <property name="text">
          <string>Show placemark</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>298</width>
           <height>44</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="m_actionBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="libmarble.qrc">
     <normaloff>:/icons/format-text-bold.png</normaloff>:/icons/format-text-bold.png</iconset>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
   <property name="toolTip">
    <string>Bold</string>
   </property>
  </action>
  <action name="m_actionItalics">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="libmarble.qrc">
     <normaloff>:/icons/format-text-italic.png</normaloff>:/icons/format-text-italic.png</iconset>
   </property>
   <property name="text">
    <string>Italics</string>
   </property>
   <property name="toolTip">
    <string>Italics</string>
   </property>
  </action>
  <action name="m_actionUnderlined">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="libmarble.qrc">
     <normaloff>:/icons/format-text-underline.png</normaloff>:/icons/format-text-underline.png</iconset>
   </property>
   <property name="text">
    <string>Underlined</string>
   </property>
   <property name="toolTip">
    <string>Underlined</string>
   </property>
  </action>
  <action name="m_actionAddImage">
   <property name="icon">
    <iconset resource="libmarble.qrc">
     <normaloff>:/icons/insert-image.png</normaloff>:/icons/insert-image.png</iconset>
   </property>
   <property name="text">
    <string>Add image</string>
   </property>
   <property name="toolTip">
    <string>Add image</string>
   </property>
  </action>
  <action name="m_actionAddLink">
   <property name="icon">
    <iconset resource="libmarble.qrc">
     <normaloff>:/icons/insert-link.png</normaloff>:/icons/insert-link.png</iconset>
   </property>
   <property name="text">
    <string>Add link</string>
   </property>
   <property name="toolTip">
    <string>Add link</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Marble::PlacemarkEditHeader</class>
   <extends>QWidget</extends>
   <header>PlacemarkEditHeader.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="libmarble.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UiEditPlacemarkDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>407</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
